pkg/mark/attachment.go

```diff
@@ -160,6 +155,22 @@ func ResolveAttachments(
 	return attaches, nil
 }
 
+func prepareAttachments(base string, replacements []string) ([]Attachment, error) {
```

Comment:
doesn't look like we need this `error` in the return variables, but it's fine tho